saas/security/saas_security.xml

```diff
@@ -8,13 +8,14 @@
   <record model="res.groups" id="group_user">
     <field name="name">User</field>
     <field name="implied_ids" eval="[(4, ref('base.group_user'))]"/>
-    <field name="users" eval="[(4, ref('base.user_root'))]"/>
+    <field name="users" eval="[(4, ref('base.user_demo'))]"/>
     <field name="category_id" ref="saas.module_management"/>
   </record>
 
   <record model="res.groups" id="group_manager">
     <field name="name">Manager</field>
     <field name="implied_ids" eval="[(4, ref('saas.group_user'))]"/>
+    <field name="users" eval="[(4, ref('base.user_admin'))]"/>
```

Comment:
Complete this entry with this: (4, ref('base.user_root'))

Reply:
If we add (4, ref('base.user_root')) to (4, ref('base.user_admin')), then average users will have saas-manager-rights

Reply:
You need to add both values: for superuser and for admin-admin